tests/integration_tests/test_sklearn.py

```diff
@@ -289,6 +289,14 @@ def test_objective_error_score_nan() -> None:
     ):
         optuna_search.fit(X)
 
+    for trial in optuna_search.study_.get_trials():
```

Comment:
`optuna_search.trials_` is unavailable because the property requires at least one complete trial.